src/main/scala/webservice/CromIamApiService.scala

```diff
@@ -14,130 +18,174 @@ trait SwaggerService extends SwaggerUiResourceHttpService {
           complete {
-            returnInternalServerError("workflow query post")
+            forwardToCromwell(req)
           }
+        }
       }
     }
 
-  def abortRoute =
+  def abortRoute: Route =
     path("api" / "workflows" / Segment / Segment / "abort") { (version, possibleWorkflowId) =>
       post {
-        complete {
-          returnInternalServerError("workflow abort")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
+          }
         }
       }
     }
 
-  def submitRoute =
+  def submitRoute: Route =
     path("api" / "workflows" / Segment) { version =>
       post {
-        complete{
-          returnInternalServerError("submit workflow")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
+          }
         }
       }
     }
 
-  def submitBatchRoute =
+  def submitBatchRoute: Route =
     path("api" / "workflows" / Segment / "batch") { version =>
       post {
-        complete {
-          returnInternalServerError("batch submit workflow")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
           }
         }
+        }
       }
 
-  def workflowOutputsRoute =
+  def workflowOutputsRoute: Route =
     path("api" / "workflows" / Segment / Segment / "outputs") { (version, possibleWorkflowId) =>
       get {
-        complete {
-          returnInternalServerError("workflow outputs")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
+          }
         }
       }
     }
 
-  def workflowLogsRoute =
+  def workflowLogsRoute: Route =
     path("api" / "workflows" / Segment / Segment / "logs") { (version, possibleWorkflowId) =>
       get {
-        complete {
-          returnInternalServerError("workflow logs")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
+          }
         }
       }
     }
 
-  def metadataRoute =
+  def metadataRoute: Route =
     path("api" / "workflows" / Segment / Segment / "metadata") { (version, possibleWorkflowId) =>
       get {
-        complete {
-          returnInternalServerError("workflow metdata")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
+          }
         }
       }
     }
 
-  def timingRoute =
+  def timingRoute: Route =
     path("api" / "workflows" / Segment / Segment / "timing") { (version, possibleWorkflowId) =>
       get {
-        complete {
-          returnInternalServerError("workflow timing")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
+          }
         }
       }
     }
 
-  def statsRoute =
+  def statsRoute: Route =
     path("api" / "engine" / Segment / "stats") { version =>
       get {
-        complete {
-          returnInternalServerError("engine stats")
+        extractRequest { req =>
+          complete {
+            forwardToCromwell(req)
```

Comment:
I don't think we want to forward any stats requests to Cromwell, we should return some kind of "no you cant do that" response directly from CromIam - https://github.com/broadinstitute/cromwell/issues/2137.